README.md

```diff
@@ -35,6 +35,14 @@ omero_ms_image_region_user: 'omero-server'
 
 - omero_ms_image_region_update_nginx: if false, it will not update the nginx config file
 
+- omero_ms_image_region_download_URL: The download URl for the distibuted ms build (zip file)
+- omero_ms_image_region_ms_sha256: The sha256 for the  for the distibuted ms build
+
+The ms build should be compatible with the installed omero-server, i.e.:
+- The bio-format version for both the microservice and Omero-Server should be the same
```

Comment:
```suggestion
- The Bio-Formats version of the microservice should match the Bio-Formats version of the OMERO.server
```
See 
https://omero.readthedocs.io/en/stable/sysadmins/unix/server-installation.html for motivation of OMERO.server spelling.